src/main/java/com/chekanova/imagetool/validation/CustomExceptionHandler.java

```diff
@@ -0,0 +1,24 @@
+package com.chekanova.imagetool.validation;
+
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+
+import java.io.IOException;
+import java.util.Date;
+import java.util.LinkedHashMap;
+import java.util.Map;
+
+@ControllerAdvice
+public class CustomExceptionHandler {
+    @ExceptionHandler({IOException.class, InterruptedException.class, IllegalArgumentException.class})
```

Comment:
Please don't add IOException.class, InterruptedException.class, because for these exceptions we shouldn't return BAD_REQUEST

Reply:
ok